doc/sphinx/reference/vcl.rst

```diff
@@ -365,6 +365,13 @@ individually:
 
   Negated ACL entries are never folded.
 
+* `+fold,noreport` - Fold without reporting
```

Comment:
I think we should avoid introducing new syntax with a comma, we already have a precedent somewhere else:

```
param.set vsl_mask +VfpAcct,+VdpAcct
```

I think it would be simpler to introduce a new `report` flag defaulting to `+report` and therefore allow to flag an ACL with `-report` to disable ACL warnings, starting folding warnings and including future warnings.